src/SystemCommands-MethodCommands/SycPushDownMethodCommand.class.st

```diff
@@ -27,13 +27,10 @@ SycPushDownMethodCommand class >> methodShortcutActivation [
 { #category : #execution }
 SycPushDownMethodCommand >> asRefactorings [
 	"Return push down method refactoring using method"
-	|dialog refactoring|
-	refactoring :=  RBPushDownMethodRefactoring 
-		pushDown: methods  
+	| refactoring |
+	refactoring := RBPushDownMethodRefactoring 
+		pushDown: (methods collect: #selector)
```

Comment:
Can you avoid to use this form?
better use (methods collect: [ :each | each selector ]